test/core.test.js

```diff
@@ -4,10 +4,54 @@ describe('ravelin.core', function() {
   });
 
   describe('#id', function() {
-    it('returns IDs that expire after idExpiryDays', function() {
+    it('can be configured with a string', function() {
+      var r = new Ravelin({
+        id: 'my-device-id'
+      });
+      return r.core.id().then(function(id) {
+        expect(id).to.equal('my-device-id');
```

Comment:
Similar to my question about `sync()`, would you need to check the deviceId is inside the cookie?

Reply:
No, not doing that here.